openstates/ny/models.py

```diff
@@ -34,20 +38,83 @@ def __init__(self, scraper, session, bill, details):
             'senate': senate_url})
 
     def _scrub_name(self, name):
-        junk = [
+        junk = (re.compile(rgx) for rgx in [
```

Comment:
I don't think the tweak of this regex flow makes the code more readable. Is it necessary to change for functionality's sake? I'd be happy to keep it how it currently is, otherwise.